README.md

```diff
@@ -1 +1,119 @@
-# React-Capstone
\ No newline at end of file
+![](https://camo.githubusercontent.com/8a4ae3fb98faf74ddf78a6677ceaa6e8872f7f340f569b7c5e1aa9bcc4061d95/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f4d6963726f76657273652d626c756576696f6c6574)
+
+# Metrics WebApp
+
+> This is a mobile web application that displays a list of metrics generated from the [Narrativa API](https://covid19tracking.narrativa.com/index_en.html). This API provides data on COVID-19.
+
+![screenshot](src/screenshots/metrics-app.png)
```

Comment:
- [ ] This link is broken. Please let's fix it.

Reply:
Done